workspace-loader/src/json-rpc/websocket-client.ts

```diff
@@ -93,12 +93,20 @@ export class WebsocketClient implements ICommunicationClient {
         }
     }
 
+    private sleep(ms) {
+        return new Promise(resolve => setTimeout(resolve, ms));
+    }
+
     /**
      * Sends pointed data.
      *
      * @param data to be sent
      */
-    send(data: any): void {
+    async send(data: any) {
```

Comment:
just add `:void` at the end of the line

Reply:
I guess we can't define `async send(data: any): void` as async function must return a value, AFAIK. But I see what you want. I'll add an information for the return value.

Reply:
Yes, indeed. It's not possible to define void for async functions. I didn't know that before.

Reply:
Modified and force-pushed.